config/owasp/suppressions.xml

```diff
@@ -25,52 +26,8 @@
     <gav regex="true">^org\.springframework\.(security|boot):(spring-security-rsa:1\.0\.1[0-1].*|spring-boot-starter-oauth2-(client|resource-server):2\.6\.[0-8])</gav>
     <cve>CVE-2022-22976</cve>
```

Comment:
@JamesCollettCGI  - CVE-2022-22976, CVE-2022-22978 and CVE-2021-22112 are false positives. Can you move these CVEs to false positive section and include the ticket reference?

Reply:
Moved CVE-2022-22976 , CVE-2022-22978 , CVE-2021-22112 to false positives with ticket references.